### PR TITLE
gh-93247: Fix assert function in asyncio locks test

### PR DESCRIPTION
**Pull request by cykerway**
Fix assert function: change `assertTrue` to `assertEqual`.

#93247

### CODE REVIEW

**Comment by oda-gitso** [comment]
#93247 (so that viewers of the issue can see that a pull request has been made).

File: Misc/NEWS.d/next/Tests/2022-05-26-07-59-05.gh-issue-93247.qOdpor.rst
Comment (kumaraditya303) [inline Misc/NEWS.d/next/Tests/2022-05-26-07-59-05.gh-issue-93247.qOdpor.rst]: News entry for such small change it not needed, you can remove this.

**Review by JelleZijlstra — APPROVED** [review]
Good catch!

Assuming CI passes, I'll leave this for a few days before merging to give the asyncio maintainers a chance to take a look.

**Pull request merged**
